### ISSUE TITLE
Question: Is this repo still active?

### ISSUE THREAD

**Issue by danielemery** [neilenns/node-red-contrib-wled2#117]
Hi there, there are a few bugs and feature improvements in both this repo and the forked one (which does not have Github issues enabled) that I would like to see addressed somewhere.

I would be interested in submitting some PRs and improving some things, I am wondering if I should:
- Work to get them into this upstream (or the v3 fork)
- Create a new fork (probably of the more-recent `node-red-contrib-wled3` fork.)
- Start a new repo from scratch

If anyone is still monitoring this repo I would love to get your input!

**Comment by danielemery** [comment]
Some examples of issues & features I am interested in (if this repo is still active I will make them separate issues):
- I would like to share connection details between WLED nodes using a configuration node
- I would like to have another node that queries WLED for the current state of the strip for use with other automations
- Currently when my WLED has a relay, for some reason the node can turn it on but not off (this does work with home assistant making the call to wled)

**Comment by neilenns** [comment]
Yes, I'm still here! Thanks for reaching out, those changes sound fun. Feel free to open individual issues for each one then a PR for each.
